asmcli/asmcli.sh

```diff
@@ -70,13 +70,16 @@ RELEASE_LINE=""
 PREVIOUS_RELEASE_LINE=""
 KPT_BRANCH=""
 NAMESPACE_EXISTS=0
+KUBECONFIG_SUPPLIED=0
```

Comment:
It seems like the couple of places that this needs to be checked, it could just be replaced with `[[ -n "${KUBECONFIG"} ]]` and we could get rid of this variable. Is there a specific reason to pull it into a global variable?

If it's just for readability, I think having a function is a bit cleaner so data isn't duplicated:

```
kubeconfig_defined() {
  if [[ -z "${KUBECONFIG}" ]]; then false; fi
}
```

Reply:
replied in other comment that **KUBECONFIG_SUPPLIED** is needed.